repo2docker/buildpacks/base.py

```diff
@@ -518,14 +556,28 @@ def get_assemble_scripts(self):
             ))
         except FileNotFoundError:
             pass
+        if self.stencila_manifest_contexts:
+            for context in self.stencila_manifest_contexts:
+                if(context == 'py' or context == 'pyjp'):
```

Comment:
Could both of these be true? If so, maybe `if {'py', 'pyjp'}.intersection(self.stencila_manafest_contexts):` to only trigger this once.

Reply:
Yes, a document can contain both `py` and `pyjp` code chunks. Neat trick, I must get more used to truthiness in Python :-).

Reply:
Regardin triggering: This would have been evaluated only once though because of the internal `self._stencila_manifest_contexts`, right? Or would that be once be class instance, not once per session?